Seulement dans ./designerPlugin: .moc
Seulement dans ./designerPlugin: .obj
Seulement dans ./examples/animation: .moc
Seulement dans ./examples/animation: .obj
Seulement dans ./examples/callback: .moc
Seulement dans ./examples/callback: .obj
Seulement dans ./examples/cameraLight: .moc
Seulement dans ./examples/cameraLight: .obj
Seulement dans ./examples/clippingPlane: .moc
Seulement dans ./examples/clippingPlane: .obj
Seulement dans ./examples/constrainedCamera: .moc
Seulement dans ./examples/constrainedCamera: .obj
Seulement dans ./examples/constrainedFrame: .moc
Seulement dans ./examples/constrainedFrame: .obj
diff -abur ../libQGLViewer-2.3.9/examples/contribs/agora/Viewer/agoraViewer.cpp ./examples/contribs/agora/Viewer/agoraViewer.cpp
--- ../libQGLViewer-2.3.9/examples/contribs/agora/Viewer/agoraViewer.cpp	2010-12-06 10:42:46.000000000 +0100
+++ ./examples/contribs/agora/Viewer/agoraViewer.cpp	2011-01-11 14:35:46.000000000 +0100
@@ -47,7 +47,7 @@
 class BoardConstraint : public Constraint
 {
 public:
-	virtual void constrainRotation(Quaternion& q, Frame * const fr)
+	virtual void constrainRotation(Quaternion& q, Frame * fr)
 	{
 		const Vec up = fr->transformOf(Vec(0.0, 0.0, 1.0));
 		Vec axis = q.axis();
diff -abur ../libQGLViewer-2.3.9/examples/contribs/blobWar/Viewer/blobWarViewer.cpp ./examples/contribs/blobWar/Viewer/blobWarViewer.cpp
--- ../libQGLViewer-2.3.9/examples/contribs/blobWar/Viewer/blobWarViewer.cpp	2010-12-06 10:42:46.000000000 +0100
+++ ./examples/contribs/blobWar/Viewer/blobWarViewer.cpp	2011-01-11 14:36:11.000000000 +0100
@@ -47,7 +47,7 @@
 class BoardConstraint : public Constraint
 {
 public:
-  virtual void constrainRotation(Quaternion& q, Frame * const fr)
+  virtual void constrainRotation(Quaternion& q, Frame * fr)
   {
     const Vec up = fr->transformOf(Vec(0.0, 0.0, 1.0));
     Vec axis = q.axis();
diff -abur ../libQGLViewer-2.3.9/examples/contribs/dvonn/dvonnviewer.cpp ./examples/contribs/dvonn/dvonnviewer.cpp
--- ../libQGLViewer-2.3.9/examples/contribs/dvonn/dvonnviewer.cpp	2010-12-06 10:42:46.000000000 +0100
+++ ./examples/contribs/dvonn/dvonnviewer.cpp	2011-01-11 14:36:40.000000000 +0100
@@ -42,7 +42,7 @@
   class BoardConstraint : public Constraint
   {
   public:
-    virtual void constrainRotation(Quaternion& q, Frame * const fr)
+    virtual void constrainRotation(Quaternion& q, Frame * fr)
       {
 	const Vec up = fr->transformOf(Vec(0.0, 0.0, 1.0));
 	Vec axis = q.axis();
Seulement dans ./examples/drawLight: .moc
Seulement dans ./examples/drawLight: .obj
Seulement dans ./examples/fastDraw: .moc
Seulement dans ./examples/fastDraw: .obj
Seulement dans ./examples/frameTransform: .moc
Seulement dans ./examples/frameTransform: .obj
Seulement dans ./examples/frustumCulling: .moc
Seulement dans ./examples/frustumCulling: .obj
Seulement dans ./examples/interface: .moc
Seulement dans ./examples/interface: .obj
Seulement dans ../libQGLViewer-2.3.9/examples/interface: ui_viewerInterface.Qt4.h
Seulement dans ./examples/keyboardAndMouse: .moc
Seulement dans ./examples/keyboardAndMouse: .obj
Seulement dans ./examples/keyFrames: .moc
Seulement dans ./examples/keyFrames: .obj
Seulement dans ./examples/luxo: .moc
Seulement dans ./examples/luxo: .obj
Seulement dans ./examples/manipulatedFrame: .moc
Seulement dans ./examples/manipulatedFrame: .obj
Seulement dans ./examples/mouseGrabber: .moc
diff -abur ../libQGLViewer-2.3.9/examples/mouseGrabber/mouseGrabber.cpp ./examples/mouseGrabber/mouseGrabber.cpp
--- ../libQGLViewer-2.3.9/examples/mouseGrabber/mouseGrabber.cpp	2010-12-06 10:42:47.000000000 +0100
+++ ./examples/mouseGrabber/mouseGrabber.cpp	2011-01-11 14:37:16.000000000 +0100
@@ -25,7 +25,7 @@
 using namespace qglviewer;
 using namespace std;
 
-void CameraPathPlayer::checkIfGrabsMouse(int x, int y, const Camera* const)
+void CameraPathPlayer::checkIfGrabsMouse(int x, int y, const Camera*)
 {
   // Rectangular activation array - May have to be tune depending on your default font size
   setGrabsMouse((x < 80) && (y<yPos()) && ((yPos()-y) < 16));
diff -abur ../libQGLViewer-2.3.9/examples/mouseGrabber/mouseGrabber.h ./examples/mouseGrabber/mouseGrabber.h
--- ../libQGLViewer-2.3.9/examples/mouseGrabber/mouseGrabber.h	2010-12-06 10:42:47.000000000 +0100
+++ ./examples/mouseGrabber/mouseGrabber.h	2011-01-11 14:38:35.000000000 +0100
@@ -26,11 +26,11 @@
 {
 public:
   CameraPathPlayer(int nb) : pathNb(nb) {};
-  void checkIfGrabsMouse(int x, int y, const qglviewer::Camera* const camera);
+  void checkIfGrabsMouse(int x, int y, const qglviewer::Camera* camera);
   int yPos() { return 25*pathNb; };
 
 protected:
-  void mousePressEvent(QMouseEvent* const, qglviewer::Camera* const camera) { camera->playPath(pathNb); };
+  void mousePressEvent(QMouseEvent*, qglviewer::Camera* camera) { camera->playPath(pathNb); };
 
 private:
   int pathNb;
Seulement dans ./examples/mouseGrabber: .obj
diff -abur ../libQGLViewer-2.3.9/examples/multiSelect/manipulatedFrameSetConstraint.cpp ./examples/multiSelect/manipulatedFrameSetConstraint.cpp
--- ../libQGLViewer-2.3.9/examples/multiSelect/manipulatedFrameSetConstraint.cpp	2010-12-06 10:42:47.000000000 +0100
+++ ./examples/multiSelect/manipulatedFrameSetConstraint.cpp	2011-01-11 14:39:31.000000000 +0100
@@ -35,7 +35,7 @@
   objects_.append(o);
 }
 
-void ManipulatedFrameSetConstraint::constrainTranslation(qglviewer::Vec &translation, Frame *const)
+void ManipulatedFrameSetConstraint::constrainTranslation(qglviewer::Vec &translation, Frame *)
 {
 #if QT_VERSION < 0x040000
   for (QPtrList<Object>::iterator it=objects_.begin(), end=objects_.end(); it != end; ++it)
@@ -45,7 +45,7 @@
     (*it)->frame.translate(translation);
 }
 
-void ManipulatedFrameSetConstraint::constrainRotation(qglviewer::Quaternion &rotation, Frame *const frame)
+void ManipulatedFrameSetConstraint::constrainRotation(qglviewer::Quaternion &rotation, Frame * frame)
 {
   // A little bit of math. Easy to understand, hard to guess (tm).
   // rotation is expressed in the frame local coordinates system. Convert it back to world coordinates.
diff -abur ../libQGLViewer-2.3.9/examples/multiSelect/manipulatedFrameSetConstraint.h ./examples/multiSelect/manipulatedFrameSetConstraint.h
--- ../libQGLViewer-2.3.9/examples/multiSelect/manipulatedFrameSetConstraint.h	2010-12-06 10:42:47.000000000 +0100
+++ ./examples/multiSelect/manipulatedFrameSetConstraint.h	2011-01-11 14:39:43.000000000 +0100
@@ -33,8 +33,8 @@
   void clearSet();
   void addObjectToSet(Object* o);
 
-  virtual void constrainTranslation(qglviewer::Vec &translation, qglviewer::Frame *const frame);
-  virtual void constrainRotation(qglviewer::Quaternion &rotation, qglviewer::Frame *const frame);
+  virtual void constrainTranslation(qglviewer::Vec &translation, qglviewer::Frame * frame);
+  virtual void constrainRotation(qglviewer::Quaternion &rotation, qglviewer::Frame * frame);
 
 private :
 #if QT_VERSION < 0x040000
Seulement dans ./examples/multiSelect: .moc
Seulement dans ./examples/multiSelect: .obj
Seulement dans ./examples/multiView: .moc
Seulement dans ./examples/multiView: .obj
Seulement dans ./examples/overpainting: .moc
Seulement dans ./examples/overpainting: .obj
Seulement dans ./examples/screenCoordSystem: .moc
Seulement dans ./examples/screenCoordSystem: .obj
Seulement dans ./examples/select: .moc
Seulement dans ./examples/select: .obj
Seulement dans ./examples/simpleViewer: .moc
Seulement dans ./examples/simpleViewer: .obj
Seulement dans ./examples/standardCamera: .moc
Seulement dans ./examples/standardCamera: .obj
Seulement dans ./examples/stereoViewer: .moc
Seulement dans ./examples/stereoViewer: .obj
Seulement dans .: libQGLViewer-2.3.9-py.patch
diff -abur ../libQGLViewer-2.3.9/QGLViewer/constraint.cpp ./QGLViewer/constraint.cpp
--- ../libQGLViewer-2.3.9/QGLViewer/constraint.cpp	2010-12-06 10:42:47.000000000 +0100
+++ ./QGLViewer/constraint.cpp	2011-01-11 14:28:06.000000000 +0100
@@ -117,7 +117,7 @@
 /*! Depending on translationConstraintType(), constrain \p translation to be along an axis or
   limited to a plane defined in the Frame local coordinate system by
   translationConstraintDirection(). */
-void LocalConstraint::constrainTranslation(Vec& translation, Frame* const frame)
+void LocalConstraint::constrainTranslation(Vec& translation, Frame* frame)
 {
   Vec proj;
   switch (translationConstraintType())
@@ -141,7 +141,7 @@
 /*! When rotationConstraintType() is AxisPlaneConstraint::AXIS, constrain \p rotation to be a rotation
   around an axis whose direction is defined in the Frame local coordinate system by
   rotationConstraintDirection(). */
-void LocalConstraint::constrainRotation(Quaternion& rotation, Frame* const)
+void LocalConstraint::constrainRotation(Quaternion& rotation, Frame*)
 {
   switch (rotationConstraintType())
     {
@@ -170,7 +170,7 @@
 /*! Depending on translationConstraintType(), constrain \p translation to be along an axis or
   limited to a plane defined in the world coordinate system by
   translationConstraintDirection(). */
-void WorldConstraint::constrainTranslation(Vec& translation, Frame* const frame)
+void WorldConstraint::constrainTranslation(Vec& translation, Frame* frame)
 {
   Vec proj;
   switch (translationConstraintType())
@@ -204,7 +204,7 @@
 /*! When rotationConstraintType() is AxisPlaneConstraint::AXIS, constrain \p rotation to be a rotation
   around an axis whose direction is defined in the world coordinate system by
   rotationConstraintDirection(). */
-void WorldConstraint::constrainRotation(Quaternion& rotation, Frame* const frame)
+void WorldConstraint::constrainRotation(Quaternion& rotation, Frame* frame)
 {
   switch (rotationConstraintType())
     {
@@ -232,14 +232,14 @@
 
 /*! Creates a CameraConstraint, whose constrained directions are defined in the \p camera coordinate
   system. */
-CameraConstraint::CameraConstraint(const Camera* const camera)
+CameraConstraint::CameraConstraint(const Camera* camera)
   : AxisPlaneConstraint(), camera_(camera)
 {}
 
 /*! Depending on translationConstraintType(), constrain \p translation to be along an axis or
   limited to a plane defined in the camera() coordinate system by
   translationConstraintDirection(). */
-void CameraConstraint::constrainTranslation(Vec& translation, Frame* const frame)
+void CameraConstraint::constrainTranslation(Vec& translation, Frame* frame)
 {
   Vec proj;
   switch (translationConstraintType())
@@ -267,7 +267,7 @@
 /*! When rotationConstraintType() is AxisPlaneConstraint::AXIS, constrain \p rotation to be a rotation
   around an axis whose direction is defined in the camera() coordinate system by
   rotationConstraintDirection(). */
-void CameraConstraint::constrainRotation(Quaternion& rotation, Frame* const frame)
+void CameraConstraint::constrainRotation(Quaternion& rotation, Frame* frame)
 {
   switch (rotationConstraintType())
     {
diff -abur ../libQGLViewer-2.3.9/QGLViewer/constraint.h ./QGLViewer/constraint.h
--- ../libQGLViewer-2.3.9/QGLViewer/constraint.h	2010-12-06 10:42:47.000000000 +0100
+++ ./QGLViewer/constraint.h	2011-01-11 14:30:39.000000000 +0100
@@ -130,7 +130,7 @@
 
     \p translation is expressed in local frame coordinate system. Use Frame::inverseTransformOf() to
     express it in the world coordinate system if needed. */
-    virtual void constrainTranslation(Vec& translation, Frame* const frame) { Q_UNUSED(translation); Q_UNUSED(frame); };
+    virtual void constrainTranslation(Vec& translation, Frame* frame) { Q_UNUSED(translation); Q_UNUSED(frame); };
     /*! Filters the rotation applied to the \p frame. This default implementation is empty (no
       filtering).
 
@@ -139,7 +139,7 @@
 
     Use Frame::inverseTransformOf() on the \p rotation Quaternion::axis() to express \p rotation in
     the world coordinate system if needed. */
-    virtual void constrainRotation(Quaternion& rotation, Frame* const frame) { Q_UNUSED(rotation); Q_UNUSED(frame); };
+    virtual void constrainRotation(Quaternion& rotation, Frame* frame) { Q_UNUSED(rotation); Q_UNUSED(frame); };
   };
 
   /*!
@@ -209,7 +209,7 @@
     /*! @name Translation constraint */
     //@{
     /*! Overloading of Constraint::constrainTranslation(). Empty */
-    virtual void constrainTranslation(Vec& translation, Frame* const frame) { Q_UNUSED(translation); Q_UNUSED(frame); };
+    virtual void constrainTranslation(Vec& translation, Frame* frame) { Q_UNUSED(translation); Q_UNUSED(frame); };
 
     void setTranslationConstraint(Type type, const Vec& direction);
     /*! Sets the Type() of the translationConstraintType(). Default is AxisPlaneConstraint::FREE. */
@@ -241,7 +241,7 @@
     /*! @name Rotation constraint */
     //@{
     /*! Overloading of Constraint::constrainRotation(). Empty. */
-    virtual void constrainRotation(Quaternion& rotation, Frame* const frame) { Q_UNUSED(rotation); Q_UNUSED(frame); };
+    virtual void constrainRotation(Quaternion& rotation, Frame* frame) { Q_UNUSED(rotation); Q_UNUSED(frame); };
 
     void setRotationConstraint(Type type, const Vec& direction);
     void setRotationConstraintType(Type type);
@@ -282,8 +282,8 @@
     /*! Virtual destructor. Empty. */
     virtual ~LocalConstraint() {};
 
-    virtual void constrainTranslation(Vec&     translation, Frame* const frame);
-    virtual void constrainRotation   (Quaternion& rotation, Frame* const frame);
+    virtual void constrainTranslation(Vec&     translation, Frame* frame);
+    virtual void constrainRotation   (Quaternion& rotation, Frame* frame);
   };
 
 
@@ -302,8 +302,8 @@
     /*! Virtual destructor. Empty. */
     virtual ~WorldConstraint() {};
 
-    virtual void constrainTranslation(Vec&     translation, Frame* const frame);
-    virtual void constrainRotation   (Quaternion& rotation, Frame* const frame);
+    virtual void constrainTranslation(Vec&     translation, Frame* frame);
+    virtual void constrainRotation   (Quaternion& rotation, Frame* frame);
   };
 
 
@@ -319,18 +319,18 @@
   class QGLVIEWER_EXPORT CameraConstraint : public AxisPlaneConstraint
   {
   public:
-    explicit CameraConstraint(const Camera* const camera);
+    explicit CameraConstraint(const Camera* camera);
     /*! Virtual destructor. Empty. */
     virtual ~CameraConstraint() {};
 
-    virtual void constrainTranslation(Vec&     translation, Frame* const frame);
-    virtual void constrainRotation   (Quaternion& rotation, Frame* const frame);
+    virtual void constrainTranslation(Vec&     translation, Frame* frame);
+    virtual void constrainRotation   (Quaternion& rotation, Frame* frame);
 
     /*! Returns the associated Camera. Set using the CameraConstraint constructor. */
     const Camera* camera() const { return camera_; };
 
   private:
-    const Camera* const camera_;
+    const Camera* camera_;
   };
 
 } // namespace qglviewer
diff -abur ../libQGLViewer-2.3.9/QGLViewer/domUtils.h ./QGLViewer/domUtils.h
--- ../libQGLViewer-2.3.9/QGLViewer/domUtils.h	2010-12-06 10:42:47.000000000 +0100
+++ ./QGLViewer/domUtils.h	2011-01-11 14:31:08.000000000 +0100
@@ -20,9 +20,8 @@
 
 *****************************************************************************/
 
-
+# include "config.h"
 #if QT_VERSION > 0x040000
-# include <QGlobal>
 # include <QDomElement>
 # include <QString>
 # include <QStringList>
Seulement dans ./QGLViewer: libQGLViewer.so
Seulement dans ./QGLViewer: libQGLViewer.so.2
Seulement dans ./QGLViewer: libQGLViewer.so.2.3
diff -abur ../libQGLViewer-2.3.9/QGLViewer/manipulatedCameraFrame.h ./QGLViewer/manipulatedCameraFrame.h
--- ../libQGLViewer-2.3.9/QGLViewer/manipulatedCameraFrame.h	2010-12-06 10:42:47.000000000 +0100
+++ ./QGLViewer/manipulatedCameraFrame.h	2011-01-11 14:31:40.000000000 +0100
@@ -119,9 +119,9 @@
     /*! @name Mouse event handlers */
     //@{
   protected:
-    virtual void mouseReleaseEvent(QMouseEvent* const event, Camera* const camera);
-    virtual void mouseMoveEvent   (QMouseEvent* const event, Camera* const camera);
-    virtual void wheelEvent       (QWheelEvent* const event, Camera* const camera);
+    virtual void mouseReleaseEvent(QMouseEvent* event, Camera* camera);
+    virtual void mouseMoveEvent   (QMouseEvent* event, Camera* camera);
+    virtual void wheelEvent       (QWheelEvent* event, Camera* camera);
     //@}
 
     /*! @name Spinning */
diff -abur ../libQGLViewer-2.3.9/QGLViewer/manipulatedFrame.h ./QGLViewer/manipulatedFrame.h
--- ../libQGLViewer-2.3.9/QGLViewer/manipulatedFrame.h	2010-12-06 10:42:47.000000000 +0100
+++ ./QGLViewer/manipulatedFrame.h	2011-01-11 14:33:09.000000000 +0100
@@ -239,11 +239,11 @@
     /*! @name Mouse event handlers */
     //@{
   protected:
-    virtual void mousePressEvent      (QMouseEvent* const event, Camera* const camera);
-    virtual void mouseMoveEvent       (QMouseEvent* const event, Camera* const camera);
-    virtual void mouseReleaseEvent    (QMouseEvent* const event, Camera* const camera);
-    virtual void mouseDoubleClickEvent(QMouseEvent* const event, Camera* const camera);
-    virtual void wheelEvent           (QWheelEvent* const event, Camera* const camera);
+    virtual void mousePressEvent      (QMouseEvent* event, Camera* camera);
+    virtual void mouseMoveEvent       (QMouseEvent* event, Camera* camera);
+    virtual void mouseReleaseEvent    (QMouseEvent* event, Camera* camera);
+    virtual void mouseDoubleClickEvent(QMouseEvent* event, Camera* camera);
+    virtual void wheelEvent           (QWheelEvent* event, Camera* camera);
     //@}
 
   public:
@@ -255,7 +255,7 @@
     /*! @name MouseGrabber implementation */
     //@{
   public:
-    virtual void checkIfGrabsMouse(int x, int y, const Camera* const camera);
+    virtual void checkIfGrabsMouse(int x, int y, const Camera* camera);
     //@}
 
     /*! @name XML representation */
@@ -268,7 +268,7 @@
 
 #ifndef DOXYGEN
   protected:
-    Quaternion deformedBallQuaternion(int x, int y, float cx, float cy, const Camera* const camera);
+    Quaternion deformedBallQuaternion(int x, int y, float cx, float cy, const Camera* camera);
 
     int action_; // Should be a QGLViewer::MouseAction, but include loop
     Constraint* previousConstraint_; // When manipulation is without Contraint.
diff -abur ../libQGLViewer-2.3.9/QGLViewer/mouseGrabber.h ./QGLViewer/mouseGrabber.h
--- ../libQGLViewer-2.3.9/QGLViewer/mouseGrabber.h	2010-12-06 10:42:47.000000000 +0100
+++ ./QGLViewer/mouseGrabber.h	2011-01-11 14:34:19.000000000 +0100
@@ -179,7 +179,7 @@
 
     See examples in the <a href="#_details">detailed description</a> section and in the <a
     href="../examples/mouseGrabber.html">mouseGrabber example</a>. */
-    virtual void checkIfGrabsMouse(int x, int y, const Camera* const camera) = 0;
+    virtual void checkIfGrabsMouse(int x, int y, const Camera* camera) = 0;
 
     /*! Returns \c true when the MouseGrabber grabs the QGLViewer's mouse events.
 
@@ -248,23 +248,23 @@
     href="../examples/mouseGrabber.html">mouseGrabber example</a> for examples.
 
     See the \c QGLWidget::mousePressEvent() and the \c QMouseEvent documentations for details. */
-    virtual void mousePressEvent(QMouseEvent* const event, Camera* const camera) { Q_UNUSED(event) Q_UNUSED(camera) };
+    virtual void mousePressEvent(QMouseEvent* event, Camera* camera) { Q_UNUSED(event) Q_UNUSED(camera) };
     /*! Callback method called when the MouseGrabber grabsMouse() and a mouse button is double clicked.
 
     See the \c QGLWidget::mouseDoubleClickEvent() and the \c QMouseEvent documentations for details. */
-    virtual void mouseDoubleClickEvent(QMouseEvent* const event, Camera* const camera) { Q_UNUSED(event) Q_UNUSED(camera) };
+    virtual void mouseDoubleClickEvent(QMouseEvent* event, Camera* camera) { Q_UNUSED(event) Q_UNUSED(camera) };
     /*! Mouse release event callback method. See mousePressEvent(). */
-    virtual void mouseReleaseEvent(QMouseEvent* const event, Camera* const camera) { Q_UNUSED(event) Q_UNUSED(camera) };
+    virtual void mouseReleaseEvent(QMouseEvent* event, Camera* camera) { Q_UNUSED(event) Q_UNUSED(camera) };
     /*! Callback method called when the MouseGrabber grabsMouse() and the mouse is moved while a
       button is pressed.
 
     This method will typically update the state of the MouseGrabber from the mouse displacement. See
     the mousePressEvent() documentation for details. */
-    virtual void mouseMoveEvent(QMouseEvent* const event, Camera* const camera) { Q_UNUSED(event) Q_UNUSED(camera) };
+    virtual void mouseMoveEvent(QMouseEvent* event, Camera* camera) { Q_UNUSED(event) Q_UNUSED(camera) };
     /*! Callback method called when the MouseGrabber grabsMouse() and the mouse wheel is used.
 
     See the \c QGLWidget::wheelEvent() and the \c QWheelEvent documentations for details. */
-    virtual void wheelEvent(QWheelEvent* const event, Camera* const camera) { Q_UNUSED(event) Q_UNUSED(camera) };
+    virtual void wheelEvent(QWheelEvent* event, Camera* camera) { Q_UNUSED(event) Q_UNUSED(camera) };
     //@}
 
   private:
diff -abur ../libQGLViewer-2.3.9/QGLViewer/qglviewer.cpp ./QGLViewer/qglviewer.cpp
--- ../libQGLViewer-2.3.9/QGLViewer/qglviewer.cpp	2010-12-06 10:42:47.000000000 +0100
+++ ./QGLViewer/qglviewer.cpp	2011-01-11 14:46:51.000000000 +0100
@@ -1174,7 +1174,7 @@
 
 This method is called when you use the QGLViewer::SELECT mouse binding(s) (default is Shift + left
 button). Use setMouseBinding() to change this. */
-void QGLViewer::select(const QMouseEvent* event)
+void QGLViewer::select(QMouseEvent* event)
 {
 	// For those who don't derive but rather rely on the signal-slot mechanism.
 	Q_EMIT pointSelected(event);
@@ -1350,7 +1350,7 @@
 	selectBuffer_ = new GLuint[selectBufferSize()];
 }
 
-void QGLViewer::performClickAction(ClickAction ca, const QMouseEvent* const e)
+void QGLViewer::performClickAction(ClickAction ca, QMouseEvent* e)
 {
 	// Note: action that need it should updateGL().
 	switch (ca)
diff -abur ../libQGLViewer-2.3.9/QGLViewer/qglviewer.h ./QGLViewer/qglviewer.h
--- ../libQGLViewer-2.3.9/QGLViewer/qglviewer.h	2010-12-06 10:42:47.000000000 +0100
+++ ./QGLViewer/qglviewer.h	2011-01-11 14:45:59.000000000 +0100
@@ -710,7 +710,7 @@
 
 	Connect this signal to your selection method or overload select(), or more probably simply
 	drawWithNames(). */
-	void pointSelected(const QMouseEvent* e);
+	void pointSelected(QMouseEvent* e);
 
 	/*! Signal emitted by setMouseGrabber() when the mouseGrabber() is changed.
 
@@ -869,7 +869,7 @@
 	GLuint* selectBuffer() { return selectBuffer_; };
 
 	public Q_SLOTS:
-		virtual void select(const QMouseEvent* event);
+		virtual void select(QMouseEvent* event);
 		virtual void select(const QPoint& point);
 
 		void setSelectBufferSize(int size);
@@ -1244,7 +1244,7 @@
 	QMap<ClickActionPrivate, QString> mouseDescription_;
 
 	void setDefaultMouseBindings();
-	void performClickAction(ClickAction ca, const QMouseEvent* const e);
+	void performClickAction(ClickAction ca, QMouseEvent* e);
 	QMap<int, MouseActionPrivate> mouseBinding_;
 	QMap<QtKeyboardModifiers, MouseActionPrivate> wheelBinding_;
 	QMap<ClickActionPrivate, ClickAction> clickBinding_;
